src/keywords.rs

```diff
@@ -931,7 +931,7 @@ pub const RESERVED_FOR_TABLE_ALIAS: &[Keyword] = &[
     Keyword::PREWHERE,
     // for ClickHouse SELECT * FROM t SETTINGS ...
     Keyword::SETTINGS,
-    // for ClickHouse SELECT * FROM t FORMAT...
+    // for ClickHouse SELECT * FROM t FORMAT... or INSERT INTO t FORMAT...
```

Comment:
```suggestion
    // for ClickHouse SELECT * FROM t FORMAT...
```
we could also remove the full description, the keyword isn't really tied to any specific syntax/dialect